content/200-concepts/200-database-connectors/03-postgresql.mdx

```diff
@@ -8,6 +8,8 @@ metaDescription: 'This page explains how Prisma can connect to a PostgreSQL data
 
 The PostgreSQL data source connector connects Prisma to a [PostgreSQL](https://www.postgresql.org/) database server.
 
+By default, the PostgreSQL connector contains a database driver responsible for connecting to your database. You can use a [driver adapter](/concepts/components/database-drivers#driver-adapters) <span class="concept"></span> to connect to your database using a Node.js database driver.
```

Comment:
I can't find it now but I remember the terminology of "Node.js driver" to be exclusive of other runtimes. Maybe we should double check that this is the terminology we want to use?

Reply:
Confirmation from Serhii: For now, all adapters happen to be Node.js drivers, but it will change in the future.